service/src/main/java/bio/terra/landingzone/stairway/flight/ProtectedDataStepsDefinitionProvider.java

```diff
@@ -20,6 +22,7 @@ public List<Pair<Step, RetryRule>> get(
       ParametersResolver parametersResolver,
       ResourceNameGenerator resourceNameGenerator,
       LandingZoneProtectedDataConfiguration landingZoneProtectedDataConfiguration) {
+    var storageHelper = new ProtectedDataAzureStorageHelper(armManagers);
```

Comment:
Do you keep it here so we can reuse it in future across different steps?

Reply:
Ah, no, this is an artifact of a previous approach where another step was consuming this helper. I've inlined it.